.husky/pre-commit

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn lint
\ No newline at end of file
+yarn lint
```

Comment:
Seems like we should run formatting before linting since formatting should in theory fix a bunch of linting warnings/errors. And the remaining linting warnings/errors that cannot be automatically resolved via formatting would just need to be manually fixed. So, the general approach would be first format to auto-resolve as many linting warnings/errors as possible and then run linting to see what warnings/errors remain that need to be manually resolved.